### PR TITLE
Get metadata from S3

### PR DESCRIPTION
**Pull request by zachthompson**
* Retrieve metadata from S3
* Remove support for uncompressed files.
* Switch to bzip2

### CODE REVIEW

**Comment by nilnilnil** [comment]
This is clever. I'm surprised we didn't think about it before :smile:.

File: lib/DDG/Meta/Data.pm [hunk 1]
```diff
@@ -34,20 +33,19 @@ unless(%ia_metadata){
...
     debug && warn "Processing metadata";
 
-    my $f = "$mdir/metadata.json";
+    my $f = "$mdir/metadata.json.bz2";
     unless($ENV{NO_METADATA_DOWNLOAD}){
         my $ua = LWP::UserAgent->new;
         $ua->timeout(5);
         $ua->default_header('Accept-Encoding' => scalar HTTP::Message::decodable());
-        my $res = $ua->mirror('https://duck.co/ia/repo/all/json', $f);
+        my $res = $ua->mirror('http://ddg-community.s3.amazonaws.com/metadata/repo_all.json.bz2', $f);
```
Comment (nilnilnil) [inline lib/DDG/Meta/Data.pm]: No https?

Comment (zachthompson) [inline lib/DDG/Meta/Data.pm]: :+1:

Comment (jbarrett) [inline lib/DDG/Meta/Data.pm]: Does that matter for this?

**Comment by jkv** [comment]
:+1: I tested this out locally and it works as expected.

**Pull request merged**